Decred Wallet/Features/Overview/OverviewViewController.swift

```diff
@@ -106,22 +106,22 @@ class OverviewViewController: UIViewController {
     }
     
     @IBAction func showAllTransactionsButtonTap(_ sender: Any) {
```

Comment:
Just remove the buttons and the tap action listeners.

Reply:
resolved in 37e2453